aurora-postgres.tf

```diff
@@ -0,0 +1,188 @@
+variable "postgres_name" {
```

Comment:
not needed

Reply:
use `var.name`

Reply:
@osterman  should we move all variables from all files into `variables.tf` for consistency?
since this is a low-level module and not part of `terraform-root-modules` (where we include variables into the main files)

Reply:
and also all outputs to `outputs.tf`